honeybee_energy/hvac/heatcool/_base.py

```diff
@@ -0,0 +1,61 @@
+# coding=utf-8
+"""Base class for DOAS HVAC systems."""
+from __future__ import division
+
+from .._template import _TemplateSystem, _EnumerationBase
+
+from honeybee._lockable import lockable
+
+import os
+
+
+@lockable
+class _HeatCoolBase(_TemplateSystem):
+    """Base class for DOAS HVAC systems.
```

Comment:
Base class for heating and cooling systems without ventilation system

Reply:
Good catch.